platform/ios/platform/ios/app/MBXViewController.m

```diff
@@ -1672,7 +1677,24 @@ - (void)selectAnOffscreenPointAnnotation {
     }
 }
 
-- (void)centerSelectedAnnotation {
+- (void)centerSelectedAnnotationWithPadding {
+    id<MLNAnnotation> annotation = self.mapView.selectedAnnotations.firstObject;
+
+    if (!annotation)
+        return;
+
+    CGPoint point = [self.mapView convertCoordinate:annotation.coordinate toPointToView:self.mapView];
+
+    // Animate, so that point becomes the the center
+    CLLocationCoordinate2D center = [self.mapView convertPoint:point toCoordinateFromView:self.mapView];
+    MLNMapCamera *camera = self.mapView.camera.copy;
+    camera.centerCoordinate = center;
+    camera.padding = UIEdgeInsetsMake(200, 50, 10, 4);
+    
+    [self.mapView setCamera:camera animated:YES];
+}
+
+- (void)centerSelectedAnnotationWithoutPadding {
```

Comment:
I know that this adjustment on the example app is not perfect.
It's been a while since I wrote some objective-c so that's why I quickly adjusted an existing entry in the feature list instead of creating a new one.

If someone has a better idea and can maybe help me out then let me know 👍🏽